EvidenceApi.Tests/V1/UseCase/EditResidentUseCaseTests.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using NUnit.Framework;
+using Moq;
+using AutoFixture;
+using EvidenceApi.V1.UseCase;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.Domain;
+using EvidenceApi.V1.Boundary.Request;
+using FluentAssertions;
+using System.Collections.Generic;
+
+namespace EvidenceApi.Tests.V1.UseCase
+{
+    public class EditResidentUseCaseTests
+    {
+        private EditResidentUseCase _classUnderTest;
+        private Mock<IResidentsGateway> _residentsGateway;
+        private static Fixture _fixture = new Fixture();
+        private Resident _foundResident;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _residentsGateway = new Mock<IResidentsGateway>();
+            _classUnderTest = new EditResidentUseCase(_residentsGateway.Object);
+        }
+
+        [Test, TestCaseSource(nameof(EditResidentRequestParameterCombinations))]
+        public void CanEditNameEmailPhoneNumberForResident(EditResidentRequest editResidentRequest)
+        {
+            SetupMocks();
+            // var editResidentRequest = new EditResidentRequest()
```

Comment:
this could be deleted

Reply:
Oh, I missed that one, thanks! Addressed.